README.md

```diff
@@ -56,7 +56,7 @@ options:
 line format:
     link_name:          String (without spaces)
     address_{in/out}:   IPv4:Port
-    bit_error_rate:     float (0.0 ... 1.0(
+    bit_error_rate:     float (0.0 ... 1.0)
```

Comment:
```suggestion
    bit_error_rate:     float (0.0 ... 1.0(
```
The open ending bracket is on purpose, as the possible interval for values does not include the upper interval boundary (i.e. must be smaller than 1.0)